docs/docs/features/prerequest-scripts.md

```diff
@@ -51,4 +51,25 @@ const sessid = altair.helpers.getCookie('sessid');
 ```js
 const res = await altair.helpers.request('GET', 'https://api.agify.io/?name=michael');
 // res => {"name":"michael","age":60,"count":41938}
-```
\ No newline at end of file
+```
+
+### Persisting your data between requests
```

Comment:
```suggestion
### Persisting data between requests
```

I think this one is clearer.

Reply:
@imolorhe what do you think about this?

Reply:
I don't really have a problem here. 😄 Ultimately I would move this into a **Use Cases** section where other use cases can be added. I'm just happy to have this information documented.